From 65a7e8b8c61343b94b975a368df5960dc2b0e7e7 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 31 Oct 2019 12:11:55 +0000
Subject: [PATCH 0897/1000] MANTIS 0055465 CUSS 1.5.1 Biometrics Interface

remove minOccurs="0" for identCode in CUSS.BIOMETRIC.XSD
for compatibility reason


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1493 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/CUSS.BIOMETRIC.XSD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/CUSS.BIOMETRIC.XSD b/SelfServiceCommon/Massai/idl/Cuss/CUSS.BIOMETRIC.XSD
index d28300c6..72679ecb 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/CUSS.BIOMETRIC.XSD
+++ b/SelfServiceCommon/Massai/idl/Cuss/CUSS.BIOMETRIC.XSD
@@ -450,7 +450,7 @@
             <xsd:documentation>Data element for command enrollPerson</xsd:documentation>
         </xsd:annotation>
         <xsd:sequence>
-            <xsd:element name="identCode" type="xsd:base64Binary" minOccurs="0">
+            <xsd:element name="identCode" type="xsd:base64Binary">
                 <xsd:annotation>
                     <xsd:documentation>Base64 encoded identification data (e.g. barcode)</xsd:documentation>
                 </xsd:annotation>
-- 
2.41.0.windows.1

